packages/neuron-wallet/src/store/WalletStore.ts

```diff
@@ -28,7 +26,7 @@ export default class WalletStore {
   constructor() {
     const idOptions: Options = {
       name: env.walletIDName,
-      encryptionKey: encryptKey,
+      encryptionKey: env.storeEncryptKey,
```

Comment:
Do we really need to encrypt files?

Reply:
Encryption or not has little effect.I think better than nothing